gradle-consistent-versions-idea-plugin/src/test/java/com/palantir/gradle/versions/intellij/GradleCacheExplorerTest.java

```diff
@@ -36,27 +37,30 @@ class GradleCacheExplorerTest {
 
     @BeforeEach
     void beforeEach() {
-        List<String> projectUrls = List.of("https://repo.maven.apache.org/maven2/", "https://jcenter.bintray.com/");
-        explorer = new GradleCacheExplorer(projectUrls);
+        explorer = new GradleCacheExplorer();
     }
 
     @Test
     void test_gets_valid_urls_only() {
+        Set<String> projectUrls = Set.of("https://repo.maven.apache.org/maven2/", "https://jcenter.bintray.com/");
+
         assertThat(explorer.isValidResourceUrl(
-                        "https://repo.maven.apache.org/maven2/com/example/artifact/1.0/artifact-1.0.pom"))
+                        projectUrls, "https://repo.maven.apache.org/maven2/com/example/artifact/1.0/artifact-1.0.pom"))
                 .as("because the URL is from a known valid repository and ends with .pom")
                 .isTrue();
 
-        assertThat(explorer.isValidResourceUrl("https://jcenter.bintray.com/com/example/artifact/1.0/artifact-1.0.jar"))
+        assertThat(explorer.isValidResourceUrl(
+                        projectUrls, "https://jcenter.bintray.com/com/example/artifact/1.0/artifact-1.0.jar"))
                 .as("because the URL is from a known valid repository and ends with .jar")
                 .isTrue();
 
-        assertThat(explorer.isValidResourceUrl("https://example.com/com/example/artifact/1.0/artifact-1.0.pom"))
+        assertThat(explorer.isValidResourceUrl(
+                        projectUrls, "https://example.com/com/example/artifact/1.0/artifact-1.0.pom"))
                 .as("because the URL is not from a known valid repository")
                 .isFalse();
 
         assertThat(explorer.isValidResourceUrl(
-                        "https://repo.maven.apache.org/maven2/com/example/artifact/1.0/artifact-1.0.txt"))
+                        projectUrls, "https://repo.maven.apache.org/maven2/com/example/artifact/1.0/artifact-1.0.txt"))
```

Comment:
Is the disparity between the apache mirror and repo1.maven.org still apparent? Is this going to cause issues when reading in the `repositories.xml`?

Reply:
from my testing the repo url that ends up in the `gcv-maven-repositrories.xml` matches the repos in the bin files